src/Sylius/Bundle/AdminBundle/Resources/config/routing/ajax/taxon.yml

```diff
@@ -4,7 +4,24 @@ sylius_admin_ajax_taxon_index:
     defaults:
         _controller: sylius.controller.taxon:indexAction
         _sylius:
-            grid: sylius_admin_taxon
+            serialization_groups: [Ajax]
+            repository:
+                method: searchByName
```

Comment:
We shoudl keep naming convention of repositories: `searchByName` -> `findByNamePart` or `findByPartOfName` ?